paymentsheet/src/main/java/com/stripe/android/paymentelement/embedded/content/EmbeddedStateHelper.kt

```diff
@@ -72,13 +72,6 @@ internal class DefaultEmbeddedStateHelper @Inject constructor(
                     "Use RowSelectionBehavior.Default or disable Google Pay and saved payment methods."
             )
         }
-        if (configuration.embeddedViewDisplaysMandateText && isRowSelectionImmediateAction) {
```

Comment:
This is a revert of https://github.com/stripe/stripe-android/pull/11111

Rather than crashing, we now show the mandate for the merchant instead.